extensions/objects/templates/literally-canvas-diagram-editor.component.ts

```diff
@@ -75,26 +81,28 @@ angular.module('oppia').component('literallyCanvasDiagramEditor', {
           'Polygon', LiterallyCanvasHelperService.polygonSVGRenderer);
         LC.defineSVGRenderer(
           'Text', LiterallyCanvasHelperService.textSVGRenderer);
-        ctrl.lc = LC.init(document.getElementById('lc'), {
-          imageSize: {
-            width: ctrl.diagramWidth, height: ctrl.diagramHeight},
-          imageURLPrefix: '/assets/literallyCanvas/img',
-          toolbarPosition: 'bottom',
-          defaultStrokeWidth: DEFAULT_STROKE_WIDTH,
-          strokeWidths: STROKE_WIDTH,
-          // Eraser tool is removed because svgRenderer has not been
-          // implimented in LiterallyCanvas. Can include once it is
-          // implimented.
-          tools: [
-            LC.tools.Pencil,
-            LC.tools.Line,
-            LC.tools.Ellipse,
-            LC.tools.Rectangle,
-            LC.tools.Text,
-            LC.tools.Polygon,
-            LC.tools.Pan,
-            LC.tools.Eyedropper
-          ]
+        angular.element(document).ready(function() {
+          ctrl.lc = LC.init(document.getElementById(ctrl.lcID), {
+            imageSize: {
+              width: ctrl.diagramWidth, height: ctrl.diagramHeight},
+            imageURLPrefix: '/third_party/static/literallycanvas-0.5.2/lib/img',
+            toolbarPosition: 'bottom',
+            defaultStrokeWidth: DEFAULT_STROKE_WIDTH,
+            strokeWidths: ALLOWED_STROKE_WIDTHS,
+            // Eraser tool is removed because svgRenderer has not been
+            // implemented in LiterallyCanvas. Can include once it is
```

Comment:
Please write all your comments using complete, full English sentences. The last sentence is a sentence fragment / seems to be missing some words.

Reply:
Done.